nomad/structs/acl.go

```diff
@@ -829,6 +829,14 @@ func (a *ACLAuthMethodConfig) Copy() *ACLAuthMethodConfig {
 	return c
 }
 
+// ACLAuthClaims is the claim mapping of the OIDC auth method in a format that
+// can be used with go-bexpr. This structure is used during rule binding
+// evaluation.
+type ACLAuthClaims struct {
+	Value any
+	List  any
```

Comment:
I think we can make these map types, so we avoid asserting it each time to grab the data.
```suggestion
	Value map[string]string{}
	List  map[string][]string{}
```

Reply:
Ah right I forgot these will always be maps 🤦‍♂️